src/client/services/cloak-service.spec.js

```diff
@@ -883,5 +1006,11 @@ describe('cloak service', () => {
 
             expect(window.cloak.message).toHaveBeenCalledWith('submitAnswer', 7);
         });
+
+        it('messages the server getRandomNumber when messageGetRandomNumber is called', () => {
+            cloakService.messageGetRandomNumber();
+
+            expect(window.cloak.message).toHaveBeenCalledWith('getRandomNumber');
```

Comment:
Cloak will eventually be removed from window. At that point we can use `__RewireAPI__.__GetDependency__('cloak?')` to get what we need. Call me over if you need an explanation :+1: